src/Sarif.Driver/Sdk/SarifLogger.cs

```diff
@@ -90,15 +91,8 @@ public SarifLogger(string outputFilePath, bool verbose, ToolInfo toolInfo, RunIn
 
         }
 
-        public SarifLogger(TextWriter writer, bool verbose, ToolInfo toolInfo, RunInfo runInfo)
+        public SarifLogger(TextWriter textWriter, bool verbose, ToolInfo toolInfo, RunInfo runInfo) : this(textWriter, verbose)
```

Comment:
@lgolding this useful change you suggested in a previous PR was zapped in an unfortunate merge incident that is still too painful for me to discuss in much detail.